source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
-- [Release 2022-02-03](#release-2022-02-03)
+
+## Release 2022-03-30
+
+### Improvements
+
+#### User Interface (UI)
+ - A new field was added to the client configuration to let clients know the database schema version of the server. The applied database migrations have also been added to the **System Console**.
```

Comment:
Depending on where this is shown in the System Console, this may not apply for Cloud

Reply:
@isacikgoz Let me know if this impacts cloud or just self-managed.

Reply:
I think cloud instances does not have access to database page on system console. At least I'd expect so :)

Reply:
Correct. Cloud instances do not include the Environment section in the System Console.